examples/forward/source_space_morphing.py

```diff
@@ -22,19 +22,17 @@
 
 # %%
 
-import os.path as op
-
 import mne
 
 data_path = mne.datasets.sample.data_path()
-subjects_dir = op.join(data_path, 'subjects')
-fname_trans = op.join(data_path, 'MEG', 'sample',
-                      'sample_audvis_raw-trans.fif')
-fname_bem = op.join(subjects_dir, 'sample', 'bem',
-                    'sample-5120-bem-sol.fif')
-fname_src_fs = op.join(subjects_dir, 'fsaverage', 'bem',
-                       'fsaverage-ico-5-src.fif')
-raw_fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+subjects_dir = (data_path / 'subjects')
```

Comment:
```suggestion
subjects_dir = data_path / 'subjects'
```

doesn't need parentheses when it all fits on one line